src/handler.rs

```diff
@@ -77,6 +81,7 @@ impl Handler {
             logger,
             mapping,
             default_handler: Box::new(NoOpHandler::default()),
+            disable_folding,
```

Comment:
Why do we need to add this to the global handler? only the InitializeHandler should need this.